diff -u ../pspell_comp.php ./pspell_comp.php
--- ../pspell_comp.php	2005-06-24 15:32:02.000000000 +0300
+++ ./pspell_comp.php	2005-06-24 15:38:15.000000000 +0300
@@ -44,9 +44,9 @@
 
 if(!defined('ASPELL_BIN')) define('ASPELL_BIN','aspell');
 
-define(PSPELL_FAST,1);         # Fast mode (least number of suggestions)
-define(PSPELL_NORMAL,2);       # Normal mode (more suggestions)
-define(PSPELL_BAD_SPELLERS,3); # Slow mode (a lot of suggestions) )
+define('PSPELL_FAST',1);         # Fast mode (least number of suggestions)
+define('PSPELL_NORMAL',2);       # Normal mode (more suggestions)
+define('PSPELL_BAD_SPELLERS',3); # Slow mode (a lot of suggestions) )
 
 function pspell_config_create($language, $spelling=null, $jargon=null, $encoding='iso8859-1'){
     return new Pspell($language, $spelling, $jargon, $encoding);
@@ -120,23 +120,23 @@
     function _prepareArgs(){
         $this->args = '';
 
-        if($this->$language){
-            $this->args .= ' --lang='.escapeshellarg($language);
-        }else{
+        if ($this->language) {
+            $this->args .= ' --lang='.escapeshellarg($this->language);
+        } else {
             return false; // no lang no spell
         }
 
         #FIXME how to support spelling?
 
-        if($this->$jargon){
-            $this->args .= ' --jargon='.escapeshellarg($jargon);
+        if ($this->jargon){
+            $this->args .= ' --jargon='.escapeshellarg($this->jargon);
         }
 
-        if($this->$encoding){
-            $this->args .= ' --encoding='.escapeshellarg($encoding);
+        if($this->encoding){
+            $this->args .= ' --encoding='.escapeshellarg($this->encoding);
         }
 
-        switch ($this->mode){
+        switch ($this->mode) {
             case PSPELL_FAST:
                 $this->args .= ' --sug-mode=fast';
                 break;
diff -u ../spell_checker.php ./spell_checker.php
--- ../spell_checker.php	2005-06-24 15:32:02.000000000 +0300
+++ ./spell_checker.php	2005-06-24 15:48:17.000000000 +0300
@@ -38,6 +38,7 @@
  *************************************************************/
 
 //user configurable list of allowed HTML tags.  Thanks to Jake Olefsky for this little addition.
+// FIXME with register_globals = on security risk (glen)
 if(empty($allowed_html)) $allowed_html = '<p><a><br><b><i><img><strong><small><ul><li>';
 
 //if pspell doesn't exist, then include the pspell wrapper for aspell
