--- spell_checker/pspell_comp.php.oryg	2007-04-19 14:37:05.000000000 +0200
+++ spell_checker/pspell_comp.php	2007-04-19 14:38:49.000000000 +0200
@@ -44,9 +44,9 @@
 
 if(!defined('ASPELL_BIN')) define('ASPELL_BIN','aspell');
 
-define(PSPELL_FAST,1);         # Fast mode (least number of suggestions)
-define(PSPELL_NORMAL,2);       # Normal mode (more suggestions)
-define(PSPELL_BAD_SPELLERS,3); # Slow mode (a lot of suggestions) )
+define('PSPELL_FAST',1);         # Fast mode (least number of suggestions)
+define('PSPELL_NORMAL',2);       # Normal mode (more suggestions)
+define('PSPELL_BAD_SPELLERS',3); # Slow mode (a lot of suggestions) )
 
 function pspell_config_create($language, $spelling=null, $jargon=null, $encoding='iso8859-1'){
     return new Pspell($language, $spelling, $jargon, $encoding);
@@ -129,11 +129,11 @@
         #FIXME how to support spelling?
 
         if($this->$jargon){
-            $this->args .= ' --jargon='.escapeshellarg($jargon);
+            $this->args .= ' --jargon='.escapeshellarg($this->jargon);
         }
 
         if($this->$encoding){
-            $this->args .= ' --encoding='.escapeshellarg($encoding);
+            $this->args .= ' --encoding='.escapeshellarg($this->encoding);
         }
 
         switch ($this->mode){
@@ -233,4 +233,4 @@
     }
 }
 
-?>
\ Brak znaku nowej linii na koñcu pliku
+?>
